docs/D-kit/Waypoint_Following/apollo_installation_cn.md

```diff
@@ -420,32 +420,31 @@ sudo docker run hello-world
 
 #### 下载Apollo源代码
 
-预装了Ubuntu18.04LTS的客户的当前home目录下已有apollo代码，可以跳过此步骤；自己安装的客户请参考以下操作来下载apollo源代码。
+a、预装了Ubuntu18.04LTS操作系统的客户的当前home目录下已有apollo代码，请按照以下步骤来更新代码：
 
-在终端中输入以下命令：
+```
+cd ~/apollo
+git checkout .
+git pull
+```
+
+b、自己安装操作系统的客户请参考以下操作来下载apollo源代码：
 
 ```
 cd ~
 sudo apt update
 sudo apt install git -y
 git init
-git clone https://github.com/ApolloAuto/apollo.git
-```
-
-代码下载的时间视网速的快慢而有所区别，请耐心等待；
-下载完成后的代码在~/apollo目录下，然后执行以下命令：
-
-```
-git checkout -b r5.5.0 origin/r5.5.0
+git clone -b r5.5.0 https://gitee.com/ApolloAuto/apollo.git
```

Comment:
Please also update gitee link of Apollo repo for the following two docs:

https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_clone_apollo_repo_from_china.md
https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_clone_apollo_repo_from_china_cn.md

Reply:
OK